chia/util/keychain.py

```diff
@@ -399,6 +406,27 @@ def delete_key_by_fingerprint(self, fingerprint: int):
             index += 1
             pkent = self._get_pk_and_entropy(get_private_key_user(self.user, index))
 
+    def delete_keys(self, keys_to_delete: List[Tuple[PrivateKey, bytes]]):
+        """
+        Deletes all keys in the list.
+        """
+        remaining_keys = {str(x[0]) for x in keys_to_delete}
+        index = 0
+        pkent = self._get_pk_and_entropy(get_private_key_user(self.user, index))
+        while index <= MAX_KEYS:
```

Comment:
if len(remaining_keys)==0 you can also exit loop, correct?

Reply:
Good catch